app/controllers/application_controller.rb

```diff
@@ -107,4 +107,20 @@ def not_authorized
     flash[:notice] = "Sorry, you are not authorized to perform this action."
     redirect_to(root_url)
   end
+
+  def check_admin_email_change_confirmation_notice(user) # for casa admins
+    notice = "#{user.role} was successfully updated."
+    if user.saved_changes.include?("unconfirmed_email")
+      notice += " Confirmation Email Sent."
+    end
+    notice
+  end
+
+  def check_all_casa_admin_email_change_confirmation_notice(user) # for all casa admins
+    notice = "#{user.role} was successfully updated."
+    if user.saved_changes.include?("unconfirmed_email")
+      notice += " Confirmation Email Sent."
+    end
+    notice
+  end
```

Comment:
These methods look exactly the same to me, are they not? Can we get rid of one and maybe make the name of the method more generic?

Reply:
You're totally right, after refactoring both of them they've basically become identical. I can remove the second one. Would something like `check_unconfirmed_email` be more appropriate?

Reply:
Sure! Can it be `check_unconfirmed_email_notice` ?

Reply:
Absolutely! I can add those changes to #4787. Thank you!